GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -202,11 +204,24 @@ private void OnGuiCreateBuildSelect()
             EditorGUILayout.Space();
             EditorGUILayout.BeginHorizontal();
             EditorGUILayout.LabelField("APK File Name", GUILayout.MinWidth(FieldMinWidth));
-            QuickDeployConfig.Config.apkFileName = EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName,
-                GUILayout.MinWidth(FieldMinWidth));
+            QuickDeployConfig.Config.apkFileName =
+                EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName, GUILayout.MinWidth(FieldMinWidth));
+            if (GUILayout.Button("Browse", GUILayout.Width(ShortButtonWidth)))
+            {
+                var defaultApkFileName = Path.GetFullPath("base.apk");
```

Comment:
I don't think this line is needed. See below.

Reply:
Resolved.